projects/storefrontapp-e2e-cypress/cypress/helpers/customer360.ts

```diff
@@ -94,7 +94,7 @@ function addProductToCart(): void {
 
 function writeReview(): void {
   checkout.goToProductDetailsPage();
-  productDetails.verifyShowReviewsLink();
```

Comment:
is `verifyShowReviewsLink ` still being used by other tests?

Reply:
Yes, used in product-detail